code/modules/SCP/SCPs/SCP-096.dm

```diff
@@ -133,7 +133,7 @@
 			continue
 		if(H in satisfied_urges)
 			continue
-		if(H.stat || H.equipment_tint_total == TINT_BLIND)
+		if(!H.can_see(src, 1)) //096 technically memetic and therefore memetic blocking apparel should blockout his face
```

Comment:
Labels it as a memetic check. Memetic checks take into account memetic insulation without having it overridden by other factors. Basically, if you're wearing something with memetic protection but something that doesn't blind you (like a scrambler headset), a check that's non memetic still returns true because you can still see something that isn't a memetic hazard while wearing only the headset. On the other hand, if the one is passed, the check becomes memetic and will return false as scramblers block out memetic hazards. This is documented somewhat in memetic_insulation.dm.

Reply:
neat